### PR TITLE
Initial release + CI

### PR DESCRIPTION
**Pull request by Rushaway**

### CODE REVIEW

File: addons/sourcemod/scripting/AutoSilencer.sp [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+bool g_bIsAutoSEnabled[MAXPLAYERS + 1] = {false, ...};
+Handle g_hCookie = INVALID_HANDLE;
+
+public Plugin myinfo =
+{
+	name = "SM Autosilencer",
+	author = "Franc1sco franug, Dolly, .Rushaway",
+	description = "AutoSilence m4a1 and usp",
+	version = "1.3.1",
+	url = "https://nide.gg"
```
Comment (maximee-leroyy) [inline addons/sourcemod/scripting/AutoSilencer.sp:20]: put the github url instead

File: addons/sourcemod/scripting/AutoSilencer.sp [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+#pragma newdecls required
+
+bool primary[2048];
+bool g_bIsAutoSEnabled[MAXPLAYERS + 1] = {false, ...};
+Handle g_hCookie = INVALID_HANDLE;
+
+public Plugin myinfo =
+{
+	name = "SM Autosilencer",
+	author = "Franc1sco franug, Dolly, .Rushaway",
```
Comment (maximee-leroyy) [inline addons/sourcemod/scripting/AutoSilencer.sp:17]: where did you find this plugin? (we can fork it if it exists on the original author github)

Comment (Rushaway) [reply addons/sourcemod/scripting/AutoSilencer.sp:17]: https://github.com/Franc1sco/AutoSilencer/tree/master

Comment (Rushaway) [reply addons/sourcemod/scripting/AutoSilencer.sp:17]: I will wait for you ping when you forked the repo, then I will open this PR again.

Comment (maximee-leroyy) [reply addons/sourcemod/scripting/AutoSilencer.sp:17]: done here:
- https://github.com/srcdslab/sm-plugin-autosilencer

Comment (maximee-leroyy) [reply addons/sourcemod/scripting/AutoSilencer.sp:17]: @Rushaway

Comment (Rushaway) [reply addons/sourcemod/scripting/AutoSilencer.sp:17]: https://github.com/srcdslab/sm-plugin-autosilencer/pull/1

**Pull request closed without merging**